### PR TITLE
Add new icons and fix router-link

### PR DESCRIPTION
**Pull request by LukasHirt**
We've added a `key` and `checklist` icons.
We've added an optional `to` prop to the `oc-tag` component so that it is properly assigned to the element in case of `router-link` type.

### CODE REVIEW

File: src/components/OcTag.vue [hunk 1]
```diff
@@ -19,6 +19,15 @@ export default {
       required: false,
       default: "span",
       validator: (type) => type.match(/(span|button|router-link|a)/)
+    },
+
+    /**
+     * Target of the router link
+     */
+    to: {
```
Comment (kulmann) [inline src/components/OcTag.vue:27]: Why is it needed, when it is added to the root component anyway?

Comment (LukasHirt) [reply src/components/OcTag.vue:27]: TBH I am not sure but if it's not passed, then in ocis-web it doesn't render with console errors about missing required `to` prop and the whole path is then undefined :shug:

File: changelog/unreleased/new-icons [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Enhancement: Add new icons
+
+We've added a `key` and `checklist` icons.
+
+https://fontawesome.com/icons/key?style=solid
+https://fontawesome.com/icons/tasks?style=solid
+https://github.com/owncloud/owncloud-design-system/pull/975
```
Comment (kulmann) [inline changelog/unreleased/new-icons:7]: The first link should always be the primary link. Could you move the PR link to the top?

File: src/components/OcTag.vue (hunk 1 above)
Comment (kulmann) [reply src/components/OcTag.vue:27]: Ok 🤷 😅

File: changelog/unreleased/new-icons (hunk 2 above)
Comment (LukasHirt) [reply changelog/unreleased/new-icons]: Done

**Review by kulmann — APPROVED** [review]
🚀

**Pull request merged**